### DISCUSSION TITLE
Why it's not able to import and use enpoise all of sudden

### DISCUSSION THREAD

**Discussion by fitbyprabhat** [neonbjb/tortoise-tts#346] [General]
---> 10 from einops import rearrange, repeat, reduce 11 from einops.layers.torch import Rearrange 12 ModuleNotFoundError: No module named 'einops'

**Comment by neonbjb** [comment]
einops is great! everyone should use it! :)

**Comment by Zavnok** [comment]
Your [issue](https://github.com/neonbjb/tortoise-tts/issues/347) has been solved.

**Reply by fitbyprabhat** [reply]
hey, can you help me i don't know why the default read.py all of sudden start crashing due to full usage of system ram in collab can this change can be reverted back?
